src/mmw/apps/home/views.py

```diff
@@ -30,9 +30,11 @@ def get_stream_layers():
     return stream_layers
 
 
-def get_client_settings():
+def get_client_settings(request):
+    EMBED_FLAG = settings.ITSI['embed_flag']
     client_settings = {
         'client_settings': json.dumps({
+            EMBED_FLAG: request.session.get(EMBED_FLAG, False),
```

Comment:
I'm just reusing the existing `embed_flag`, so it is relatively easy to change (although not really, since we explicitly define it on the Backbone side). Effectively, this will look like:

```javascript
{
    itsi_embed: true,
    ...
}
```